example/android/kotlinlib/1-hello-kotlin/build.mill

```diff
@@ -27,10 +29,45 @@ object app extends AndroidAppKotlinModule {
   def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
 
   // Configuration for ReleaseKey
+  override def releaseKeyPath = root
   def androidReleaseKeyName: T[String] = Task { "releaseKey.jks" }
   def androidReleaseKeyAlias: T[String] = Task { "releaseKey" }
   def androidReleaseKeyPass: T[String] = Task { "MillBuildTool" }
   def androidReleaseKeyStorePass: T[String] = Task { "MillBuildTool" }
+
+  /* TODO this won't be needed once the debug keystore is implemented */
+  private def mainRoot = root
+
+  object test extends AndroidAppKotlinTests with TestModule.Junit4 {
+    def ivyDeps = super.ivyDeps() ++ Agg(
+      ivy"junit:junit:4.13.2"
+    )
+  }
+
+  object it extends AndroidAppKotlinIntegrationTests with AndroidTestModule.AndroidJUnit {
+    def repositoriesTask = Task.Anon {
+      super.repositoriesTask() ++
+        Seq(MavenRepository("https://maven.google.com"))
+    }
+    def kotlinVersion = "2.0.20"
+    def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
```

Comment:
Should these be inherited from the enclosing `AndroidAppKotlinModule` by default? That's what we do in the unit tests above, which is why we do not need to set these configs there